### PR TITLE
[DNSAPI] add dns_rcode0.sh - Support for https://my.rcodezero.at/api-doc

### PR DESCRIPTION
**Pull request by MikeAT**
This PR implements the dnsapi for https://www.rcodezero.at based on their API Documentation available under  https://my.rcodezero.at/api-doc

### CODE REVIEW

**Comment by Neilpang** [comment]
please check this guide first
https://github.com/Neilpang/acme.sh/wiki/DNS-API-Dev-Guide

**Comment by MikeAT** [comment]
I did some code polish. Did I miss any other point from the Dev guide?

File: dnsapi/dns_rcode0.sh [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    _err "Please login and create your token at httsp://my.rcodezero.at/enableapi and try again."
+    return 1
+  fi
+
+  if [ -z "$RCODE0_TTL" ]; then
+    RCODE0_TTL="$DEFAULT_RCODE0_TTL"
+  fi
+
+  #save the api addr and key to the account conf file.
+  _saveaccountconf_mutable RCODE0_URL "$RCODE0_URL"
```
Comment (Neilpang) [inline dnsapi/dns_rcode0.sh]: don't save the RCODE0_URL  all the time.
We should only save it if user set a non-default value.

**Comment by Neilpang** [comment]
RCODE0_URL  and RCODE0_TTL  is missing from the usage wiki page.

please also make sure it suport wildcard cert and all these format:
```
_acme-challenge.www.example.com
_acme-challenge.example.com
_acme-challenge.example.co.uk
_acme-challenge.www.example.co.uk
_acme-challenge.sub1.sub2.www.example.co.uk
```

File: dnsapi/dns_rcode0.sh (hunk 1 above)
Comment (MikeAT) [reply dnsapi/dns_rcode0.sh]: fixed in bc04475

**Comment by MikeAT** [comment]
> RCODE0_URL and RCODE0_TTL is missing from the usage wiki page.

I added them to the wiki

> please also make sure it suport wildcard cert and all these format:
> 
> ```
> _acme-challenge.www.example.com
> _acme-challenge.example.com
> _acme-challenge.example.co.uk
> _acme-challenge.www.example.co.uk
> _acme-challenge.sub1.sub2.www.example.co.uk

These (+wildcards) are supported

**Pull request merged**